upload_form.py

```diff
@@ -459,8 +550,11 @@ def set_columns(filename, fileformat=None):
                                                     branch=branch_database,
                                                    )
     except Exception as ex:
-        return render_template('error.html', error=str(ex),
-                               traceback=traceback.format_exc(ex))
+        if testmode:
+            raise ex
+        else:
+            return render_template('error.html', error=str(ex),
```

Comment:
Yeah, that wasn't very clear. The output is expected to be of the form to contain 2 items:
```
        link_pull_database, link_pull_uploads = \
                create_pull_request(username=username,
                                    merged_table=merged_table,
                                    merged_table_name=merged_table_name,
                                    table_widths=table_widths,
                                    unique_filename=unique_filename,
                                    testmode=testmode)
``` 
If the return for an error is:
```
            return render_template('error.html', error=str(ex),
                                   traceback=traceback.format_exc(ex))
```
A ValueError for "Too many values to unpack" is raised instead of creating the actual error page. The ```, None``` I meant would just get tacked on after the ```render_template```.